hikari/commands.py

```diff
@@ -132,10 +132,13 @@ class CommandOption:
     r"""The command option's name.
 
     !!! note
-        This will match the regex `^[\w-]{1,32}$` in Unicode mode and will be
+        This will match the regex `^[-_\p{L}\p{N}\p{sc=Deva}\p{sc=Thai}]{1,32}$` in Unicode mode and will be
         lowercase.
     """
 
+    description_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict, kw_only=True)
```

Comment:
Move down with `name_localizations` and 
```suggestion
    description_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = attr.field(factory=dict)
```